cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -68,12 +72,14 @@ import qualified Shelley.Spec.Ledger.LedgerState as Shelley
 import           Cardano.Api.Address
 import           Cardano.Api.Block
 import           Cardano.Api.Certificate
-import           Cardano.Api.SerialiseRaw
 import           Cardano.Api.Eras
 import           Cardano.Api.KeysShelley
 import           Cardano.Api.Modes
 import           Cardano.Api.NetworkId
-import           Cardano.Api.ProtocolParameters
+import           Cardano.Api.Orphans ()
+import           Cardano.Api.ProtocolParameters (GenesisParameters, ProtocolParameters,
+                   ProtocolParametersUpdate, fromShelleyGenesis, fromShelleyPParams,
+                   fromShelleyProposedPPUpdates)
```

Comment:
I suggest we stick with the style of not using explicit imports within the API modules for other API modules.

Reply:
That was HLS getting a little too excited.